src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -202,8 +202,8 @@ module InternalUse =
             "Borrow" 
             "Return"
 
-            "Data"
-            "Item"
+            "Data" // REL0920: can be removed once we give errors for underscore patterns
```

Comment:
Are we using `REL𝑚𝑚𝑦𝑦` to indicate deprecation timers? That seems like a really good practice to apply uniformly across repos, perhaps even to have a CI check for.

Reply:
Yes, exactly. I started with TODO, and then though better of it and it seems useful to have tags ready that we can easily search for when we already know certain things need to be done with release X.

Reply:
FYI, I've copied this pattern here:
https://github.com/microsoft/iqsharp/blob/23ef42c4ea5b872b12a35dcd494845b4dc18bfb3/src/Core/Workspace/Project.cs#L233

In that case it's only a guess as to which future release, but it's definitely a better reminder than `TODO`.